token/js/package.json

```diff
@@ -52,6 +52,7 @@
         "@types/eslint-plugin-prettier": "^3.1.0",
         "@types/mocha": "^9.1.0",
         "@types/node": "^16.11.21",
+        "@types/node-fetch": "^2.6.2",
```

Comment:
Added ```node-fetch``` as i was not able to build the lib getting error for not having node-fetch as it was imported in ```@solana/web3.js```